data/src/main/scala/ch.epfl.scala.index.data/cleanup/GithubRepoExtractor.scala

```diff
@@ -73,7 +73,8 @@ class GithubRepoExtractor extends DefaultJsonProtocol {
         matches(m, s"$groupId $artifactId")
       }.map(_._2).toList
 
-    (fromPoms ++ fromClaims).map{ 
+    /* use claims first - because project indexing uses only the head github repo */
```

Comment:
we could have multiple project reference in one project, this would handle old repos.